contents/copy/cfs/selection-process.md

```diff
@@ -0,0 +1,11 @@
+## Selection Process
+
+1. All submissions are anonymized, so that there is no bias towards the submitter.
```

Comment:
No comma, also maybe "to ensure there is no"

Reply:
please edit?